src/as2fm/as2fm_common/logging.py

```diff
@@ -76,12 +76,13 @@ def _assemble_message(severity: Severity, element: "lxml.etree._Element", messag
         "The element must have a filepath attribute. This is set by "
```

Comment:
I have to admit that the implementation of this function slipped completely my attention! If I knew it existed, I would have started using it so long ago! :)